cli/src/main.rs

```diff
@@ -78,6 +79,11 @@ fn run() -> Result<()> {
         .long("quiet")
         .short("q");
 
+    let apply_all_captures_arg = Arg::with_name("apply-all-captures")
+        .help("Apply all captures to highlights")
+        .long("apply-all-captures")
+        .short("apply-all");
```

Comment:
Short usually a one char length parameter and if I remember correctly latest clap version accept only single chars here. I wouldn't assign short name here at all.

Reply:
Yeah I added that a while ago before refactoring the cli, removed